metrics/internal/cache/pv.go

```diff
@@ -94,6 +112,48 @@ func runCephRBDStatus(config *cephMonitorConfig, pool, image string) (Clients, e
 	return clients, err
 }
 
+func runCephRBDChildrenCount(config *cephMonitorConfig, pool, namespace, image string) (int, error) {
+
+	if config.monitor == "" && config.id == "" && config.key == "" {
```

Comment:
Not related to this PR. This check seems to be wrong it should be `or` check not `and`